css/index.css

```diff
@@ -64,17 +68,74 @@ div .box {
 
 
 }
-
-
-
-
-
-
-
-
-
-
-
+.container {
+    
+    
+}
+body .firstParagragh{
+    border-bottom: 2px dotted;
+    padding: 20px 20px;
+    width: 104%;
+    margin-left: -2%;
+    line-height:1.5;
+}
+.box1{
```

Comment:
For each individual box, let the parent style most of the individual box style then just override what you want to change.  On these boxes it would just be the background color.  The display: flex and justify-content: space-around only needs to be set 1 time in the parent, flexbox nests 2 deep.